### PR TITLE
New feature for demo

### PR DESCRIPTION
**Pull request by mattdelliott**
@rbrtkrgr

### CODE REVIEW

**Comment by rbrtkrgr** [comment]
Oh, I see.

File: test.txt [hunk 1]
```diff
@@ -1,3 +1,5 @@
 sadlkfsad;lkfjsdal;kfjsldk;fjlk;
 
-a new line
\ No newline at end of file
+a new line
+
+lkdsjlksdjflkjsdf
```
Comment (rbrtkrgr) [inline test.txt]: This doesn't make sense to me, but whatevs.

**Comment by mattdelliott** [comment]
:+1:

**Comment by rbrtkrgr** [comment]
:+1:

**Pull request merged**